musicbrainz/graphql.go

```diff
@@ -0,0 +1,201 @@
+// This file is part of the go-meta library.
+//
+// Copyright (C) 2017 JAAK MUSIC LTD
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+// If you have any questions please contact yo@jaak.io
+
+package musicbrainz
+
+import (
+	"database/sql"
+	"errors"
+
+	"github.com/ipfs/go-cid"
```

Comment:
please consider explicitly aliasing to `cid` for clarity

Reply:
The package name is already `cid`?

Reply:
I mean:

```
import (
    cid "github.com/ipfs/go-cid"
)
```

Reply:
I'm not sure why we would want to "alias" the `cid` package to `cid` when it is already called `cid`?

Reply:
If my eyes are not deceiving me, it says `go-cid` not `cid`.

It may be fair go-style to assume automatic truncation of `[-]go[-]` in the package name. But I find it unclear.

Reply:
Ok sorry I think I missed the point you were making.

I don't personally think it is worth the work to alias a package with the same name if the last part of the URL doesn't match, because I think imports should just be managed by `goimports`, references resolved using `godef`, and aliasing reserved for when you actually need to alias something (e.g. importing two packages named `math` for example).

That being said, I don't mind doing so if it is helpful to others, so perhaps we should make that change across the codebase in a separate PR?